libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -18,6 +18,8 @@ userdata Location method get_bearing float Location
 userdata Location method get_distance_NED Vector3f Location
 userdata Location method get_distance_NE Vector2f Location
 userdata Location method change_alt_frame boolean Location::AltFrame'enum Location::AltFrame::ABSOLUTE Location::AltFrame::ABOVE_TERRAIN
+userdata Location method copy Location
```

Comment:
If we wanted to keep this in line with the rest of the generator this should be:
```suggestion
userdata Location copy
```
This require add a special case keyword to the parser at a higher level.

The difference is where the code fails. For example as it is the generator will accept:
`userdata Location method copy Vector3f`

The code will not build of course. Generally the binding generator should fail in cases where invalid code would be generated. Of course we have not covered all edge cases. But in general the generator will give a useful error and line number in the description file.

Reply:
that requires much more complex changes to scripting, and I suspect more flash space. We can protect against someone using the wrong type by code review. We don't protect against the wrong type for other bindings

Reply:
The output of the binding generator would be the same. 

I'm not saying this is a bad way to do do it. Just that it does not fit with the way other special cases are handled.